src/inspector/view/qml/MuseScore/Inspector/notation/lines/internal/LineStyleSection.qml

```diff
@@ -70,7 +70,7 @@ Column {
 
     FlatRadioButtonGroupPropertyView {
         id: styleSection
-        titleText: qsTrc("inspector", "Style")
+        titleText: qsTrc("inspector", "Line Style")
```

Comment:
It's part of our design style that we don't use "Title Case" (i.e. capitalising every word). Instead, we prefer "Sentence case" everywhere.
```suggestion
        titleText: qsTrc("inspector", "Line style")
```